tools/SourceKit/CMakeLists.txt

```diff
@@ -234,6 +238,11 @@ macro(add_sourcekit_executable name)
       set_target_properties(${name}
         PROPERTIES
         LINK_FLAGS "-Wl,-exported_symbol,_main")
+
+      if(SWIFT_ANALYZE_CODE_COVERAGE)
+        set_property(TARGET "${name}" APPEND_STRING PROPERTY
```

Comment:
set_target_properties overwrites LINK_FLAGS. If we can remove the set_target_properties call without changing something important then we can remove the extra code coverage configuration.